resources/com/mesosphere/global/terraform-file-dcos-terraform-test-examples/aws-0.2.x/main.tf

```diff
@@ -40,6 +40,44 @@ module "dcos" {
 
   dcos_variant              = "${var.dcos_variant}"
   dcos_license_key_contents = "${var.dcos_license_key_contents}"
+
+  additional_windows_private_agent_ips       = ["${concat(module.winagent.private_ips)}"]
+  additional_windows_private_agent_passwords = ["${concat(module.winagent.windows_passwords)}"]
+}
+
+module "winagent" {
+  source  = "dcos-terraform/windows-instance/aws"
+  version = "~> 0.2.1"
+
+  providers = {
+    aws = "aws"
+  }
+
+  cluster_name           = "${random_id.cluster_name.hex}"
+  hostname_format        = "%[3]s-winagent%[1]d-%[2]s"
+  aws_subnet_ids         = ["${module.dcos.infrastructure.vpc.subnet_ids}"]
+  aws_security_group_ids = ["${aws_security_group.test_dotnet_sample.id}", "${module.dcos.infrastructure.security_groups.admin}"]
+  aws_key_name           = "${module.dcos.infrastructure.aws_key_name}"
+  aws_instance_type      = "m5.xlarge"
+
+  # provide the number of windows agents that should be provisioned.
+  num = "1"
+}
+resource "aws_security_group" "test_dotnet_sample" {
+  name        = "dcos-test_dotnet_sample"
+  description = "Allow incoming traffic to .NET application"
+  vpc_id      = "${module.dcos.infrastructure.vpc.id}"
+
+  ingress {
+    from_port   = "${var.dotnet_sample_port}"
+    to_port     = "${var.dotnet_sample_port}"
+    protocol    = "tcp"
+    cidr_blocks = ["${data.http.whatismyip.body}/32"]
+  }
```

Comment:
thats what public agents are for. marathon-lb label are already set so lets use public agent load balancing to access the application. This also allows to see if the communication between public and private agents is working properly

Reply:
fixed at https://github.com/dcos-terraform/jenkins-library/pull/10/commits/5f3624c996c85b4ec6076f6b9d193eb8be2e100b